knexfile.js

```diff
@@ -0,0 +1,44 @@
+// Update with your config settings.
+
+module.exports = {
```

Comment:
You've got a warning in your console about `useNullAsDefault`. Remember to include it when using SQLite!